content/en/docs/concepts/policy/dynamic-admission-controllers/_index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Dynamic Admission Controllers
+content_type: concept
+weight: 40
+description: Dynamic Admission Controllers run outside the API server as separate applications that register to receive webhooks and enable complex validation, or custom mutation, based on API requests. For example, a dynamic admission controller can lookup data from OCI registries and perform image verification checks. [More...](/docs/concepts/dynamic-admission-controllers/)
```

Comment:
Don't put Markdown in descriptions (it doesn't render everywhere you might want it to)

Reply:
Ok. So, replace this: `[More...](/docs/concepts/dynamic-admission-controllers/)`?  Is there another way to specify?